cmd/monaco/internal/test/cmd.go

```diff
@@ -0,0 +1,67 @@
+/*
+ * @license
+ * Copyright 2024 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package test
+
+import (
+	"fmt"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/runner"
+	"github.com/spf13/afero"
+	"regexp"
+	"strings"
+)
+
+type Cmd struct {
+	cmd string
+	fs  afero.Fs
+}
+
+func Monacof(cmd string, args ...any) *Cmd {
```

Comment:
Please add documentation to all public symbols where it's not obvious what it does. 
Especially Monacof and Cmd are not clear; WithFs, Run, etc are clear